src/main/scala/akka/persistence/jdbc/journal/dao/ByteArrayJournalDao.scala

```diff
@@ -49,6 +50,18 @@ trait BaseByteArrayJournalDao extends JournalDaoWithUpdates {
   import journalConfig.daoConfig.{ batchSize, bufferSize, logicalDelete, parallelism }
   import profile.api._
 
+  val logger = LoggerFactory.getLogger(this.getClass)
+
+  // This logging may block since we don't control how the user will configure logback
+  // We can't use a Akka logging neither because we don't have an ActorSystem in scope and
+  // we should not introduce another dependency here.
+  // Therefore, we make sure we only log a warning for logical deletes once
+  lazy val logWarnAboutLogicalDeletionDeprecation = {
+    logger.warn(
+      "Logical deletion of events is deprecated and will be removed in akka-persistende-jdbc version 4.0.0." +
```

Comment:
Add a space at the end of the string. Other wise the concatentation will be something like ".. version 4.0.0.To disable ..."